locales/de/calendar.json

```diff
@@ -0,0 +1,48 @@
+{
+  "date": "Datum",
+  "time": "Uhrzeit",
+  "event": "Formel 1 Grand Prix Veranstaltungen",
+  "notice": {
+    "text": "Die ersten acht Rennen des revidierten Kalenders unten. Times are TBC along with additional races.",
+    "link": "Mehr lesen"
+  },
+  "badges": {
+    "tbc": "Unbestätigt",
+    "tickets": "Karten",
```

Comment:
Tickets would be more clear instead of Karten and is commonly enough used in german.